compiler/hash-semantics/src/passes/resolution/pats.rs

```diff
@@ -273,6 +273,10 @@ impl<E: SemanticEnv> ResolutionPass<'_, E> {
                 Pat::Lit(LitPat(Node::create_at(Lit::Int((*int_lit).into()), origin))),
                 origin,
             ),
+            ast::Lit::Byte(byte_lit) => Node::create_at(
+                Pat::Lit(LitPat(Node::create_at(Lit::Int((*byte_lit).into()), origin))),
```

Comment:
I wonder how we can remember that this was a ASCII literal rather than an integer in TIR...

Reply:
Perhaps for nicer formatting

Reply:
Yeah this would be nice, we can add a flag to int lit, and then use the node origin to just print the raw literal?

Reply:
Maybe make an issue for this?

Reply:
https://github.com/hash-org/hashc/issues/1001